src/main/java/com/frameworkium/core/ui/tests/BaseTest.java

```diff
@@ -191,6 +192,13 @@ public static WebDriverWrapper getDriver() {
         return driver.get().getDriver();
     }
 
+    /**
+     * @return the {@link Driver} instance for the requesting thread
+     */
+    public static Driver getFrameworkDriver() {
+        return driver.get();
```

Comment:
Don't like the idea of exposing more of our private parts in [the] public [API] unless we have to.
If we really do need to tear down the driver could we just expose a tear down method instead of giving access to the `Driver` object?